haystack/nodes/retriever/text2sparql.py

```diff
@@ -16,7 +18,12 @@ class Text2SparqlRetriever(BaseGraphRetriever):
     """
 
     def __init__(
-        self, knowledge_graph, model_name_or_path, top_k: int = 1, use_auth_token: Optional[Union[str, bool]] = None
+        self,
+        knowledge_graph: BaseKnowledgeGraph,
+        model_name_or_path: str,
+        model_version: Optional[str] = None,
```

Comment:
Let's add a line in the docstring for this param

Reply:
Right, totally forgot - thanks Branden